src/Capgemini.PowerApps.Deployment/Capgemini.PowerApps.Deployment.nuspec

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+  <metadata>
+    <id>Capgemini.Xrm.Deployment.PackageDeployer</id>
+    <version>3.0.6</version>
```

Comment:
I would revert this to `0.1.0` or something so that it's obvious that versioning is handled by GitVersion. You'll also need to remember to tag master with `v3.0.6` so that GitVersion has a baseline.